### PR TITLE
Revised some indent, typo and added function of printing in color.

### PR DESCRIPTION
**Pull request by yunminjin2**
Our 20-2-SKKU-OSS Team has studied your code. However, we had revised some mistakes.
We thought that it would be easy to see if there could be divided in color. For example, time, fuction name, code, return, variable text could be divided in different color. 

I hope it would be helpful!

Thank you.
Pysnooper_color

### CODE REVIEW

File: pysnooper/tracer.py [hunk 1]
```diff
@@ -322,7 +325,7 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         elapsed_time_string = pycompat.timedelta_format(duration)
         indent = ' ' * 4 * (thread_global.depth + 1)
         self.write(
-            '{indent}Elapsed time: {elapsed_time_string}'.format(**locals())
+            '\033[33m'+'{indent}Elapsed time: {elapsed_time_string}'+'\033[0m'.format(**locals())
```
Comment (alexmojaki) [inline pysnooper/tracer.py:328]: `format` doesn't work anymore. Just get rid of the `+`.

Colour must always be optional. Some terminals will not be able to support it so users should be able to turn it off. In particular Windows won't support this without colorama.

**Comment by cool-RR** [comment]
Guys, you clearly worked hard on this and you have a desire to contribute. But you're not careful enough and not communicative enough. Please remember that we are volunteers who are reviewing your code in our spare time.

You sent us a diff that's 80% irrelevant changes so it's difficult to find your contributions. You didn't discuss the design with us before doing all this work. You made puzzling changes like [this one](https://github.com/cool-RR/PySnooper/pull/208/files#diff-135e1eff7775caad9989e18de4dafe0a6b9f0077c7b5a4246e26ffd3e0a6f75fL484) which lower the code quality for no good reason. You changed indentation of existing lines, committed IDE-specific code which should stay out of the repo, etc.

As Alex said, color must be optional and we must be careful not to cause problem on any platforms.

Right now the way you're contributing is causing a bigger burden on us than just implementing the features ourselves. Either change your approach completely or please find another project to contribute to.

**Comment by yunminjin2** [comment]
I'm sorry about giving you confusion. Actually we're just students learning Open Source Project for first time. For a study, we have selected your projects.  Because we're not professional software engineer, there would be a lot of problems... I agree. Don't care a lot of our Pull&Request. Sorry for bothering you.

However, it was a great project to study. We could learn a lot like pull&request, commit, version control, considering different platform. Thank you for giving us a chance to study with. 

Have a good day!

**Comment by laike9m** [comment]
> I'm sorry about giving you confusion. Actually we're just students learning Open Source Project for first time. For a study, we have selected your projects. Because we're not professional software engineer, there would be a lot of problems... I agree. Don't care a lot of our Pull&Request. Sorry for bothering you.
> 
> However, it was a great project to study. We could learn a lot like pull&request, commit, version control, considering different platform. Thank you for giving us a chance to study with.
> 
> Have a good day!

This is very unprofessional and disrespectful behavior. If you want to be software engineers in the future, please don't do this. This kind of PR is pure spam.

**Pull request closed without merging**